### PR TITLE
CI: Use Travis wait improved to avoid timeout for OSX builds

### PR DESCRIPTION
**Pull request by sciunto**
## Checklist

- [Docstrings for all functions](https://github.com/numpy/numpy/blob/master/doc/example.py)
- Gallery example in `./doc/examples` (new features only)
- Benchmark in `./benchmarks`, if your changes aren't covered by an
  existing benchmark
- Unit tests
- Clean style in [the spirit of PEP8](https://www.python.org/dev/peps/pep-0008/)

## For reviewers

- Check that the PR title is short, concise, and will make sense 1 year
  later.
- Check that new functions are imported in corresponding `__init__.py`.
- Check that new features, API changes, and deprecations are mentioned in
      `doc/release/release_dev.rst`.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -123,6 +123,7 @@ before_install:
         apt-key adv --list-public-keys --with-colons | grep '^pub' | cut -d':' -f 5 | egrep -o '.{8}$' | grep -wo 762E3157 > /dev/null 2>&1 || sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 762E3157;
       fi
     - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
+        pip3 install travis-wait-improved;
```
Comment (hmaarrfk) [inline .travis.yml:126]: @stefanv @jni @emmanuelle 

This is exactly the kind of frustration that our users experience when they try to install our software.

They attempt to use:


```
pip install scikit-image
```
but then there is always some edge case.

Is this something we want to document?

If so, is a "troubleshooting" section appropriate?

**Comment by sciunto** [comment]
@scikit-image/core This time, I'm confident that this PR turns our CI back to green.

**Review by rfezzani — APPROVED** [review]
Thank you for @sciunto :wink:

File: .travis.yml (hunk 1 above)
Comment (emmanuelle) [reply .travis.yml:126]: A troubleshooting section could be a useful addition indeed.

**Comment by emmanuelle** [comment]
Woohoo, green CI ! Thank you @sciunto :-). Merging!

**Pull request merged**